testsuites/CBLTester/upgrade_tests/conftest.py

```diff
@@ -436,6 +440,11 @@ def params_from_base_suite_setup(request):
 
     utils_obj = Utils(base_url)
 
+    need_sgw_admin_auth = (not disable_admin_auth) and sync_gateway_version >= "3.0"
+    print("==============================================")
```

Comment:
Clean up prints

Reply:
cleaned up the print statement, and added proper log_info